.gitignore

```diff
@@ -24,6 +24,9 @@ var/
 .installed.cfg
 *.egg
 
+#Testing
+testing.py
```

Comment:
This should not be in the commit.

Reply:
Sorry, that was careless of me. I've removed both now